pillar/heroku/xpro.sls

```diff
@@ -96,6 +99,9 @@
       'MITXPRO_BASE_URL': 'https://xpro.mit.edu',
       'MITXPRO_SECURE_SSL_HOST': 'xpro.mit.edu',
       'ENABLE_ORDER_RECEIPTS': True,
+      'FEATURE_COURSE_DROPDOWN': True,
+      'FEATURE_WEBINARS': True,
+      'FEATURE_ENABLE_BLOG': True,
```

Comment:
Just adding it here won't have any effect on the application configuration, so you will need to add it to the YAML block below this Jinja section. If the intent is to turn it on for all environments then you can delete these Jinja values and only add it in the YAML section.